resources/sass/_base.scss

```diff
@@ -165,77 +163,77 @@ $caret-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5
   // Other styles
   var,
   var a {
-    color: $variable-type-color;
+    color: var(--variable-type-color);
     font-style: normal;
   }
 
   dt:hover var,
   dt:hover var a {
-    color: $variable-type-color-hover;
+    color: var(--variable-type-color-hover);
   }
 
   dfn {
     font-style: normal;
     font-family: monospace;
-    color: $variable-name-color;
+    color: var(--variable-name-color);
   }
 
   pre {
-    color: $text-color;
-    margin: 0 0 0 ($spacing * 3px);
+    color: var(--text-color);
+    margin: 0 0 0 calc(var(--spacing) * 3);
     padding: 5px;
     overflow-y: hidden;
     border-top: 0;
-    border: $border;
-    background: $main-background;
+    border: var(--border);
+    background: var(--main-background);
     display: block;
     word-break: normal;
   }
 
   .kint-access-path-trigger,
   .kint-folder-trigger,
   .kint-search-trigger {
-    background: rgba($text-color, 0.8);
+    background: color-mix(in srgb, var(--text-color) 80%, transparent);
     border-radius: 3px;
-    height: $color_size * 1px;
-    font-size: $color_size * 1px;
+    height: var(--color-size);
+    font-size: var(--color-size);
     margin-left: 5px;
     font-weight: bold;
-    width: $color_size * 1px;
+    width: var(--color-size);
     text-align: center;
     float: right !important;
     cursor: pointer;
-    color: $main-background;
+    color: var(--main-background);
     position: relative;
     overflow: hidden;
 
     // Carefully balanced on multiple browsers
-    line-height: $color_size * 1.1 * 1px;
+    line-height: calc(--color-size * 1.1);
```

Comment:
Broken. Missing `var()`

Reply:
In my tests you can just set this to `var(--color-size)` and it will work now

Reply:
I believe using a `flex` solution is better than relying on `line-height`, that's an outdated method.

Reply:
Actually it could just be `line-height: 1` since we set the font-size to `--color-size` anyway (sorry for nitpick heh)

Reply:
Sure, I set it to `lint-height: normal` and also applied the flex to make sure it is properly centered, check the new commit to see

(Also don't worry, I have the exact same feelings regarding the nitpicks, even that 1px makes my OCD go creazy 😂 so it'll be awesome to make sure it looks fine without any pixel displacements, whatsoever)